hackathon_site/event/jinja2/event/landing.html

```diff
@@ -27,292 +29,296 @@
                 </h5>
             </div>
+            {% if localtime(now) > localtime(registration_open_date) %}
             <div class="row">
-                <a class="btn-large waves-effect waves-light colorBtn">Login/Apply</a>
+                <a class="btn-large waves-effect waves-light secondaryColor coloredBtn">
+                    {% if localtime(now) > localtime(registration_close_date) %}
+                    Login
+                    {% else %}
+                    Login/Apply
+                    {% endif %}
+                </a>
             </div>
+            {% endif %}
             <br /><br />
         </div>
-        <div class="col s12 m5 landingImg">
-            <img src="{{ static('event/images/landing.svg') }}" alt="landing"/>
-        </div>
     </div>
-</div>
 
-<div class="container">
-    <div class="section scrollspy" id="about">
-        <div class="row">
-            <div class="col m12 l7" id="aboutText">
-                <h3 class="center">About us</h3>
+    <div class="container">
+        <div class="section scrollspy" id="about">
+            <div class="row">
+                <div class="col m12 l7">
+                    <h3 class="center white-text">About us</h3>
 
-                <p class="center">
-                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
-                    ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint
-                    occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
-                </p>
-                <p class="center">
-                    Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto
-                    beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi
-                    nesciunt.
-                </p>
-            </div>
+                    <p class="center white-text">
+                        Join us at NewHacks to transform your ideas into reality! In this 24-hour event, you and your team will make a project from scratch using software and present your project to a panel of industry
+                        judges to compete for prizes. This year, the theme for NewHacks is
+                        <span class="lightText">digital education</span> and <span class="lightText">life convenience</span> - so be creative with software solutions to improve online learning or quality of life!
```

Comment:
![image](https://user-images.githubusercontent.com/26036279/91667382-3ca99580-ead2-11ea-8470-ac64bc05d6d2.png)

This emphasis color doesn't stand out very much, at least not with the positioning of things on my screen.